programs/uxd/src/error.rs

```diff
@@ -74,7 +74,11 @@ pub enum UxdError {
     InvalidDepositoryCollateral,
     #[msg("The provided depository shares does not match the depository's one.")]
     InvalidDepositoryShares,
-    #[msg("The provided mercurial vault does not match the Depository's one.")]
+    #[msg("The Profits beneficiary provided does not match the depository's one.")]
+    InvalidProfitsBeneficiary,
+    #[msg("The Profits beneficiary provided does not match the depository's one.")]
+    ProfitsBeneficiaryNotInitialized,
+    #[msg("The profits beneficiary hasn't setup for this depository")]
```

Comment:
The profits beneficiary hasn't setup for this depository
> 
The profits beneficiary hasn't been setup for this depository

Reply:
grammar 🥲